systems/node/controller/pb/controller.proto

```diff
@@ -0,0 +1,39 @@
+syntax = "proto3";
+package ukama.node.controller.v1;
+option go_package = "github.com/ukama/ukama/systems/node/controller/pb/gen";
+
+import "validator.proto";
+
+
+// Defines the service for controller operations
+service ControllerService {
+  rpc RestartSite(RestartSiteRequest) returns (RestartSiteResponse);
+  rpc RestartNode(RestartNodeRequest) returns (RestartNodeResponse);
+}
+
+
+message RestartSiteResponse {
+  RestartStatus status = 1;
+
+}
+
+message RestartNodeRequest {
+    string nodeId = 1 [(validator.field) = {uuid_ver: 4, string_not_empty: true}, json_name = "node_id"];;
+   }
+
+message RestartSiteRequest {
+    string siteName =1;
+    string networkId = 2 [(validator.field) = {uuid_ver: 4, string_not_empty: true}, json_name = "network_id"];
+
+}
+
+message RestartNodeResponse {
+    RestartStatus status = 1;
+
+   }
+
+enum RestartStatus {
+    RESTART_STATUS_UNKNOWN = 0;
```

Comment:
I think status here should be accpeted or not.
May be other things you have here can be events from node to mesh